prime-router/src/main/kotlin/common/Environment.kt

```diff
@@ -43,6 +43,19 @@ enum class Environment(
      */
     val baseUrl: String get() = getBaseUrl(url)
 
+    /**
+     * Available feature flags for enabling different features
+     */
+    enum class FeatureFlags(
+        val enabledByDefault: Boolean = false
+    ) {
+        FHIR_ENGINE_TEST_PIPELINE();
+
+        fun enabled(): Boolean {
+            return enabledByDefault || System.getenv(this.toString()) == "enabled"
```

Comment:
I like this, but I think we we add a new one we would need specific enabled by default.  I would recommend maybe using the 0 or 1 values, or even using a comparison that ignores case.